### PR TITLE
Update for uframe changes

### PR DESCRIPTION
**Pull request by birdage**
@DanielJMaher @oceanzus changes for the uframe ...76 port, also fixed the `new_end_time` i didnt see it?

the https://github.com/asascience-open/ooi-ui-services/commit/c8c9ebca2342873aaaf0d91308055e37087e95a2 change, ignores the last row of data as it contains some content we dont need, and now we are using `pk`>`stream` as stream name is not available on the ...76 port version...

### CODE REVIEW

File: ooiservices/app/uframe/data.py [hunk 1]
```diff
@@ -84,24 +84,27 @@ def get_data(stream, instrument, yfields, xfields, include_time=True):
     # xdata = OrderedDict()
     # ydata = OrderedDict()
 
-    x = OrderedDict({k: np.empty(len(data)) for k in xfields})
-    y = OrderedDict({k: np.empty(len(data)) for k in yfields})
+    x = OrderedDict({k: np.empty(len(data)-1) for k in xfields})
+    y = OrderedDict({k: np.empty(len(data)-1) for k in yfields})
```
Comment (DanielJMaher) [inline ooiservices/app/uframe/data.py]: This is only if we were looking at 12575, but we are looking at 12576.

Comment (birdage) [inline ooiservices/app/uframe/data.py]: @DanielJMaher i have seen this on 12576 as well.....

**Comment by birdage** [comment]
@DanielJMaher @oceanzus can we close this then?

**Comment by birdage** [comment]
@DanielJMaher bump

File: ooiservices/app/uframe/data.py [hunk 2]
```diff
@@ -84,24 +84,27 @@ def get_data(stream, instrument, yfields, xfields, include_time=True):
...
-            # y
-            for yfield in yfields:
-                if yfield == 'time':
-                    y[yfield][ind] = float(row['pk']['time'])
-                else:
-                    y[yfield][ind] = row[yfield]
+        #skip the last row
+        if ind < len(data)-1:
+            # used to handle multiple streams
+            if row['pk']['stream'] == stream:
```
Comment (Bobfrat) [inline ooiservices/app/uframe/data.py]: @birdage This can stay right?

Comment (birdage) [inline ooiservices/app/uframe/data.py]: @Bobfrat you will need to add this, as we check the pk stream, which didnt exist in the old version of uframe services

**Comment by birdage** [comment]
@DanielJMaher going to close this as we dont need it now

**Pull request closed without merging**